docs/developer_guide/compute.md

```diff
@@ -250,7 +250,7 @@ import starsCount from './starsCount'
       showToast(string`The repos have ${starsCount} stars`, 5000, 'success')
     ],
     catch: new Map([
-      [HttpProviderError, showToast(string`Error: ${props`error.message`}`, 5000)]
+      [HttpProviderError, showToast(string`Error: ${props`error.body.message`}`, 5000)]
```

Comment:
`error.message` should be correct. 
See https://github.com/cerebral/cerebral/blob/master/packages/cerebral-provider-http/src/HttpProviderError.js